packages/textarea/src/styled.tsx

```diff
@@ -7,15 +9,63 @@ import {getValueWithUnit} from '@smashing/theme'
 type InputProps = TextareaProps &
   Required<Pick<TextareaProps, 'borderRadius' | 'appearance' | 'full'>>
 
-export const Textarea = styled(Text)<InputProps>`
+export const TextareaContainer = styled.div<InputProps>`
   border: none;
   border-radius: ${_ => getValueWithUnit(_.borderRadius)};
-  padding: 8px 10px;
+  padding: ${_ => (_.appearance === 'outline' ? '8px 10px' : 0)};
   box-sizing: border-box;
-  min-height: 40px;
-  resize: vertical;
+  width: fit-content;
+  margin-bottom: 0;
+  display: block;
   ${_ => ({
-    width: _.width ? getValueWithUnit(_.width) : _.full ? '100%' : undefined
+    width: _.width ? getValueWithUnit(_.width) : _.full ? '100%' : undefined,
+    color: _.theme.colors.border.active,
+    '&[aria-invalid="true"]': {
+      color: _.theme.colors.border.danger
+    }
   })}
   ${_ => getTextInputStyle(_.appearance)};
 `
+
+export const Textarea = styled(Text)`
+  height: 100%;
+  border: none;
+  background-color: transparent;
+  scrollbar-width: none;
+  -ms-overflow-style: none;
+  outline: none;
+  padding: 0;
+  resize: none;
+  ::-webkit-scrollbar {
+    display: none;
+  }
+`
+
+export const ScrollbarContainer = styled(Scrollbars).attrs(
+  ({appearance}) =>
+    appearance === 'outline' && {
```

Comment:
`default` appearance is not too different from `outline` appearance, yet it has different scrollbar - seems weird. You should create a prop `scrollbarAppearance` that will control it.